src/MobileDelivery/MobileConfirm.js

```diff
@@ -3,50 +3,36 @@ import moment from 'moment';
 import './Mobile.css';
 
 class MobileConfirm extends React.Component {
-    onSubmit(e){
-        e.preventDefault();
-        var data = {
-            raSignature: e.target.signature.value,
-            raPrintName: e.target.print.value,
-            timeCompleted: moment().format()
-        };
-        this.props.nextStep();
-        this.props.saveValues(data);
-    }
-
-    componentDidMount(){
-    }
-
     render() {
         return (
             <div className="mobile-confirmation-container">
                 <div className="mobile-confirm-card">
                     <div className="mobile-card-line"></div>
                     <p className="ms-header">Completion Summary</p>
-                    <p id="confirm-time">{moment(this.props.currentDelivery.timeDelivered).calendar()}</p>
+                    <p id="confirm-time">{moment.unix(this.props.deliveryObj.deliveredInfo.timestamp).calendar()}</p>
```

Comment:
The data in the database should be in milliseconds, not seconds. Therefore, we should be using
```
moment(this.props.deliveryObj.deliveredInfo.timestamp).calendar();
```
I figure this is because the test data you've put into firebase is in seconds. If the actual data is coming through in seconds then that's a separate discussion I guess.

Reply:
Thanks for pointing out! yea I referred to the `schema.txt` when I was working on this, was not very sure if it got updated or which one to use for production.

Reply:
Oh, I see, that makes sense. Would you mind putting little comments next to those fields in the schema indicating `// ms` and include it in with this PR? Probably my mistake for not making that clear when I was making schema changes.